src/types/condition_opcodes.py

```diff
@@ -6,14 +6,15 @@ class ConditionOpcode(bytes, enum.Enum):
     UNKNOWN = bytes([49])
     AGG_SIG = bytes([50])
     CREATE_COIN = bytes([51])
-    ASSERT_COIN_CONSUMED = bytes([52])
+    ASSERT_ANNOUNCEMENT = bytes([52])
     ASSERT_MY_COIN_ID = bytes([53])
     ASSERT_RELATIVE_TIME_EXCEEDS = bytes([54])
     ASSERT_BLOCK_INDEX_EXCEEDS = bytes([55])
     ASSERT_BLOCK_AGE_EXCEEDS = bytes([56])
     AGG_SIG_ME = bytes([57])
     ASSERT_FEE = bytes([58])
     ASSERT_TIME_EXCEEDS = bytes([59])
+    CREATE_ANNOUNCEMENT = bytes([60])
```

Comment:
Maybe switch these two so the `CREATE`s are together?

Reply:
I didn't want to renumber anything else as renumbering opcodes is a task that we've scheduled for people to have opinions on

Reply:
Assert announcement is a close analog to assert_coin_consumed